packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -293,12 +293,13 @@ export async function editSDKConnection(
     }
   });
 
-  await SDKConnectionModel.updateOne(
+  const updatedConnection = await SDKConnectionModel.findOneAndUpdate(
```

Comment:
I prefer not using `findOneAndUpdate`.  Instead, just return `{...connection, ...updates}`.